project/Environment.scala

```diff
@@ -21,7 +21,7 @@ object Environment {
     Properties.envOrElse(environmentVariable, default)
 
   lazy val hadoopVersion  = either("SPARK_HADOOP_VERSION", "2.2.0")
-  lazy val sparkVersion   = either("SPARK_VERSION", "1.4.1")
+  lazy val sparkVersion   = either("SPARK_VERSION", "1.5.0")
```

Comment:
We should probably depend on 1.5.1 or 1.5.2. EMR only supports 1.5.0 as of right now but I'm there should not be API breaks with minor version.